zproject/sentry.py

```diff
@@ -0,0 +1,23 @@
+import sentry_sdk
+from sentry_sdk.integrations import Integration
+from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.redis import RedisIntegration
+from sentry_sdk.integrations.sqlalchemy import SqlalchemyIntegration
+
+from .config import PRODUCTION
+
+
+def setup_sentry(dsn: Optional[str], *integrations: Integration) -> None:
+    if not dsn:
+        return
+    sentry_sdk.init(
+        dsn=dsn,
+        environment="production" if PRODUCTION else "development",
+        integrations=[
+            DjangoIntegration(),
+            RedisIntegration(),
+            SqlalchemyIntegration(),
+            *integrations,
+        ],
+        send_default_pii=True,
```

Comment:
I feel like we are likely to just just need user IDs, not email/name; should we think about changing this?

Reply:
We can certainly start off sending less information rather than more, yeah.